cmd/troubleshoot/cli/root.go

```diff
@@ -58,6 +58,10 @@ from a server that can be used to assist when troubleshooting a Kubernetes clust
 	cmd.Flags().Bool("allow-insecure-connections", false, "when set, do not verify TLS certs when retrieving spec and reporting results")
 	cmd.Flags().MarkHidden("allow-insecure-connections")
 
+	// `no-uri` references the `followURI` functionality where we can use an upstream spec when creating a support bundle
+	// This flag makes sure we can also disable this and fall back to the default spec.
+	cmd.Flags().Bool("no-uri", false, "Forces the support bundle to revert back to the default and not use a URI for a spec when used.`")
```

Comment:
I think the only nit I have here is in the usage docs, I might emphasize the outcome instead of language like "revert to the default" since someone might not know what the default behaviour was, and I think the "new" default behaviour is that we *will* follow a URI.

Maybe something like "When set to `true`, do not attempt to retrieve the bundle referenced by the `uri:` field"

Reply:
Thanks Ada & Xav!

Reply:
@adamancini changed this, thanks for reviewing!